src/FiscalYears/FiscalYearForm/FiscalYearForm.js

```diff
@@ -43,8 +45,16 @@ const FiscalYearForm = ({
   pristine,
   submitting,
 }) => {
+  const ky = useOkapiKy();
+
   const closeForm = useCallback(() => onCancel(), [onCancel]);
 
+  const validateName = useCallback(
+    (value) => validateFYName(ky, initialValues.id, value),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
IMHO, there is more harm from using that comment, since there is no track of changed dependencies